test/integration/nighthawk_test_server.py

```diff
@@ -78,6 +78,7 @@ class _TestCaseWarnErrorIgnoreList(
 
             # Logged for normal termination, not really a warning.
             "caught ENVOY_SIGTERM",
+            "internal_address_config is not configured. The existing default behaviour will trust RFC1918 IP addresses, but this will be changed in next release. Please explictily config internal address config as the migration step.",
```

Comment:
Did we try to find a way to explicitly configure this behavior? If we ignore it here, it will stop working in a non-obvious way in one of the future updates.

What would it cost to fix it now?

Reply:
Yes, we have to add a field to the HTTPConnectionManagerConfig instances. I will do so in a separate PR. A quick search thorugh nighthawk codebase shows we have ~5 configs so I'm not sure if we have to change just one or all.

Reply:
Thank you, please open a Github issue to track it and add a TODO next to that override, so we don't lose track of it.